app/views/account/profile/show.html.slim

```diff
@@ -0,0 +1,84 @@
+- set_meta_tags title: "#{@profile.full_name} - Overview",
+                description: "#{@profile.full_name} info"
+.content-wrapper
+  .content-header.row
+    .content-header-left.col-md-6.col-12.mb-2
+      h3.content-header-title.mb-0
+        = @profile.full_name
+      .row.breadcrumbs-top
+        .breadcrumb-wrapper.col-12
+          = render_breadcrumbs({ 'Dashboard' => account_root_path,
+                                 'Profile' => nil })
+    .content-header-right.text-md-right.col-md-6.col-12
+      = link_to edit_account_profile_path,
+                class: 'button btn-icon btn btn-grey-blue btn-danger-2 btn-round'
+        i.fa.fa-pencil-square-o
+  .content-body
+    section.row
+      .col-sm-12
+        #overview.card
+          .card-header
+            h4.card-title Profile
+          .card-content
+            .card-body
+              .row
+                .card-image.col-sm-2
+                  = image_tag(@profile.avatar, :size => 120)
+                .card-text.col-sm-8
+                  .row
+                    .card-text.col-sm-3
+                      h6 User-email:
+                    .card-text.col-sm-9
+                      = @profile.email
+                  .row
+                    .card-text.col-sm-3
+                      h6 Phone number:
+                    .card-text.col-sm-9
+                      = @profile.phone
+                  .row
+                    .card-text.col-sm-3
+                      h6 Date of birth:
+                    .card-text.col-sm-9
+                      = @profile.date_of_birth
```

Comment:
output data or birth in format like "June 4, 1960"

using localication helper (I18n.l date) + check if present